grafana-plugin/src/containers/RemoteSelect/RemoteSelect.tsx

```diff
@@ -62,17 +68,22 @@ const RemoteSelect = inject('store')(
 
     const [options, setOptions] = useReducer(mergeOptions, []);
 
-    useEffect(() => {
-      makeRequest(href, {}).then((data) => {
-        setOptions(getOptions(data.results || data));
-      });
+    const loadOptionsCallback = useCallback((query?: string): void => {
+      (async () => {
+        try {
+          const data = await makeRequest(href, { params: { search: query } });
+          const options = getOptions(data.results || data);
+          setOptions(options);
+        } catch (e) {
+          if (axios.isAxiosError(e) && e.response.status === 403 && requiredUserAction) {
```

Comment:
Is `axios.isAxisError(e)` really necessary here? Checking for status and user permission should be enough

Reply:
[`axios.isAxiosError`](https://github.com/axios/axios/blob/2a71f49bc6c68495fa419003a3107ed8bd703ad0/index.d.ts#L508) is a type guard, otherwise typescript is going to complain about me trying to access `e.response.status`